flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolInteractionsTest.java

```diff
@@ -69,33 +62,18 @@
 import static org.junit.Assert.fail;
 
 /**
- * Tests for the SlotPool using a proper RPC setup.
+ * Tests for the SlotPoolImpl interactions.
  */
-public class SlotPoolRpcTest extends TestLogger {
-
-	private static RpcService rpcService;
-
-	private static final Time timeout = Time.seconds(10L);
+public class SlotPoolInteractionsTest extends TestLogger {
```

Comment:
Is it possible to avoid concurrency in this test case?

Reply:
I think it still somewhat required because the test involve timeouts, and the simple direct main thread executor will always run into a violation if it is entered through the scheduled executor of the timeout.

Reply:
Alright, thanks for the clarification.